Dream Catchers/Assets/_Game/Scripts/_GameScripts/AI/FSM.cs

```diff
@@ -47,8 +47,11 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-
-        _currentState.Execute();
+        if (Game_Manager.instance.currentGameState == Game_Manager.GameState.PLAY)
```

Comment:
Maybe turn the "Game_Manager.instance.currentGameState == Game_Manager.GameState.PLAY"
into a function call under game manager. Just cause this call will be made a bunch.
i.e. Game_Manager.IsPaused()